Earthdawn-4E/ED4E.json

```diff
@@ -0,0 +1,8 @@
+{
+    "html": "ED4E.html",
+    "css": "ED4E.css",
+    "authors": "Brett B.",
+    "roll20userid": "Dougansf",

```

Comment:
This is supposed to be the user ID number for your Roll20 account. You can find this in your profile URL. I'm going to add the sheet, but remove this line for now since this isn't correct and I don't know what your account is.